source/extensions/filters/network/ext_authz/ext_authz.h

```diff
@@ -54,6 +54,7 @@ class Config {
       : stats_(generateStats(config.stat_prefix(), scope)),
         failure_mode_allow_(config.failure_mode_allow()),
         include_peer_certificate_(config.include_peer_certificate()),
+        retain_http_status_on_grpc_deny_(config.retain_http_status_on_grpc_deny()),
```

Comment:
This is set here but I don't think it is ever used. What would it even mean for the network filter? My guess is that this only applies to http ext_auth.

Reply:
Friendly ping